test/repository_test.rb

```diff
@@ -90,6 +90,33 @@
             UserRepository.find(@user1.id).must_equal(@user1)
           end
         end
+
+        describe 'when entity is not persisted' do
+          before do
+            @unpersisted_user = User.new(name: 'My', age: '23')
```

Comment:
there are existing test cases for `.persist` that uses different fixture, can you please consolidate all of them so that all use your newly fixture?